tags/plugins/plugins.py

```diff
@@ -0,0 +1,35 @@
+from talon import Module, actions, ui
+
+mod = Module()
+
+mod.tag("plugins", "Enable commands and contexts for apps that use optional plugins")
+
+
+@mod.scope
+def scope():
+    return {"plugin_installed": actions.user.plugins_list()}
```

Comment:
We're thinking `actions.user.plugins_list` should have a VScode exclusive implementation for now. For other variants, these could be added by folks.

Long term, we're thinking to explore getting this via the command server.

Reply:
I've looked into this and there isn't a simple way from the command server to know this folder path. But we can return a list of all the extensions from the command server.